de.schmidhuberj.tubefeeder.json

```diff
@@ -8,14 +8,13 @@
     ],
     "command": "tubefeeder",
     "finish-args": [
+        "--share=ipc",
         "--socket=fallback-x11",
         "--socket=wayland",
         "--socket=pulseaudio",
         "--device=dri",
         "--share=network",
         "--filesystem=xdg-data/tubefeeder:create",
```

Comment:
why do you need this?

Reply:
This fixes the warning `finish-args-x11-without-ipc`. Don't really know what it is for though.

Reply:
it's necessary for X11 (or fallback). It should have been noted back then at review time, must have fallen through the cracks, the linter make this a hard error.